Makefile

```diff
@@ -6,99 +6,99 @@ help:
 
 .PHONY: build
 build: ## Build the development server
-	@docker-compose build --pull
+	@docker compose build --pull
 
 .PHONY: start
 start: ## Start the development server
-	@docker-compose up -d --build
+	@docker compose up -d --build
 
 .PHONY: api-test
 api-test: ## Test the backend
-	@docker-compose run --rm api pytest --no-cov-on-fail --cov --create-db -vv
+	@docker compose run --rm api poetry run pytest --no-cov-on-fail --cov --create-db -vv
 
 .PHONY: api-lint
 api-lint: ## Lint the backend
-	@docker-compose run --rm api sh -c "black --check . && flake8"
+	@docker compose run --rm api sh -c "poetry run black --check . && poetry run flake8"
 
 .PHONY: api-bash
 api-bash: ## Shell into the backend
-	@docker-compose run --rm api bash
+	@docker compose run --rm api bash
 
 .PHONY: api-shell_plus
 api-shell_plus: ## Run shell_plus
-	@docker-compose run --rm api python ./manage.py shell_plus
+	@docker compose run --rm api poetry run ./manage.py shell_plus
 
 .PHONY: api-dev-server
 api-dev-server: ## Start backend dev server
-	@docker-compose stop api
-	@docker-compose run --user root --use-aliases --service-ports api bash -c 'pip install pdbpp && python ./manage.py runserver 0.0.0.0:8000'
+	@docker compose stop api
+	@docker compose run --user root --use-aliases --service-ports api bash -c 'pip install pdbpp && poetry run ./manage.py runserver 0.0.0.0:8000'
 
 .PHONY: makemigrations
 makemigrations: ## Make django migrations
-	@docker-compose run --rm api python ./manage.py makemigrations
+	@docker compose run --rm api poetry run ./manage.py makemigrations
 
 .PHONY: migrate
 migrate: ## Migrate django
-	@docker-compose run --rm api python ./manage.py migrate
+	@docker compose run --rm api poetry run ./manage.py migrate
 
 .PHONY: dbshell
 dbshell: ## Start a psql shell
-	@docker-compose exec db psql -Umysagw
+	@docker compose exec db psql -Umysagw
 
 .PHONY: caluma-test
 caluma-test: ## test caluma config and extensions
-	@docker-compose exec -T caluma poetry run python manage.py check
-	@docker-compose exec -T -u root caluma poetry install --no-root
-	@docker-compose exec -T caluma ./caluma/ci/test.sh
+	@docker compose exec -T caluma poetry run python manage.py check
+	@docker compose exec -T -u root caluma poetry install --no-root
+	@docker compose exec -T caluma ./caluma/ci/test.sh
 
 .PHONY: caluma-lint
 caluma-lint: ## lint caluma extensions
-	@cd ./caluma && black --check .
-	@cd ./caluma && flake8
+	@cd ./api && poetry run bash -c "cd ../caluma && black --check ."
+	@cd ./api && poetry run bash -c "cd ../caluma && flake8"
```

Comment:
I don't understand why all the `cd`-ing back and forth

Reply:
Because I want to use the `pyproject.toml` from within `./api`, but need to run the commands from  within `./caluma`.

This makes sure we use the same versions of our formatters and linters across all our python code in the project.